hangb.rb

```diff
@@ -0,0 +1,262 @@
+  end
+
+
+# ------------------
+# Check answer
+
+# m: "U, girl. Here's the method definition for get_guess. This is how I think it would be easiest for you
+#      to return an answer to the user..."
+# U: Aiite, go.
+# m: " Okay,
+#     1st - until that answer line equals the answer from shuffle.pop: draw the board, and keep
+#           displaying the answer line so they'll know how awesome they're doing. Politely ask
+#           for a letter while those conditions are true as well.  
+#     2nd - If someone/thing tries to give you more than one letter, scold them!
+#     3rd - When the user gives you a letter as a guess, evaluate that input against @answer.
+#           That's an array though, so you'll have to .split the empty spaces from the input 
+#           to make sure it looks exactly like the letters of the chosen @answer.
+#     4th - accept the input letter(s) in lower or uppercase; but since all the answers are written in upper-
+#           case, turn the user input into uppercase before you evaluate it against the @answer."
+#     Still with me?
+#  U: yup.
+#  m: "awesome, let me break down this conditional loop for you below."
+ 
+
+
+  def get_guess
+
+    until @answer_line.join == @answer
+
+      puts "Guess a letter, please:"
+      input = gets.chomp.upcase
+
+      while input.length != 1
+        puts "You're better than this. Please enter ONE letter, thanks."
+        input = gets.chomp.upcase
+      end
+
+      letters = @answer.split("")
+      # puts letters
+
+#  m: " Now some additional conditions. With ALLLL that information (lines 183 - 191):
+
+      if letters.include?(input)            # does the input match the "letters" ["include?"]
+        letters.each do |letter|            # if so, take each letter that fits the condition [ "||" ]
+          if letters.include?(input)
+            position = letters.index(input) # let's define position as the index spot in the  
+            letters[position] = "@"         # @answer array. in that spot where input letter(s)
+            @answer_line[position] = input  # match the @answer array,  put an "@." This is a temporary
+          end                               # placeholder in the @answer array so that no letters shift
+        end                                 # or change location. Now match that input (letter(s)) to the
+      else                                  # same position in the @answer_line array." 
+       
+
+        @wrong_count += 1                   # m: "U, everytime they guess wrong, add 1."
+      end
+
+      draw_board                            # m: "Right or wrong guess, draw the board that            
+      puts @answer_line.join                # corresponds with @wrong_count, and display 
+    end                                     # @answer_line as a string"
+
+# m: "Universe, when they win, that is when @answer_line.join == @answer, put the following:
+
+      puts "YOU WIN!!! The Federation thanks you!!!".red.blink    
```

Comment:
You should consider moving any drawing to a method so your game logic code is by itself